foundation/foundation/src/main/res/values/styles.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 
     <style name="Base.Theme.Vitamin" parent="Base.Theme.DayNight.Vitamin">
         <item name="colorPrimary">?attr/vtmnBackgroundBrandPrimary</item>
-        <item name="colorPrimaryDark">@color/vtmn_blue600</item>
+        <item name="colorPrimaryDark">?attr/vtmnBackgroundPrimary</item>
         <item name="colorOnPrimary">?attr/vtmnContentPrimaryReversed</item>
         <item name="colorSecondary">?attr/vtmnBackgroundAccent</item>
         <item name="colorOnSecondary">?attr/vtmnContentPrimaryReversed</item>
         <item name="colorSurface">?attr/vtmnBackgroundPrimary</item>
         <item name="colorOnSurface">?attr/vtmnContentPrimary</item>
         <item name="colorError">?attr/vtmnContentNegative</item>
         <item name="colorOnError">?attr/vtmnContentPrimaryReversed</item>
-        <item name="android:windowBackground">?attr/vtmnBackgroundPrimary</item>
-        <item name="colorControlActivated">?attr/colorSecondary</item>
+        <item name="android:windowBackground">?attr/vtmnBackgroundSecondary</item>
```

Comment:
Warning here. In setting the background at secondary, all the screens are now "grey" and not white anymore. It has strange behavior with the filled text inputs. Do we need a fix here?
![Capture d’écran 2022-01-20 à 15 42 41](https://user-images.githubusercontent.com/46402524/150360617-c386913d-6d15-414d-83bc-bd65a7bf6ab8.png)